lib/env/commands/list_types.rb

```diff
@@ -40,6 +40,7 @@ def run
           Table.emit do |t|
             headers 'Name', 'Summary'
             Env::Type.each do |t|
+              next if t.hidden
```

Comment:
This hides them from the list output, but the `type` still exists in the program, and will work if the user tries to launch an environment of that type. Is that the desired behaviour here?

Reply:
> I could do the same change for `flight desktop` as well

That's a good point to mention, actually. The behaviour in Flight Desktop is currently to just hide it in the `avail` list. I'd say keep consistent between the two, and ask what @sierra-tango-echo thinks.